### PR TITLE
allows users to comment on portfolio page, added doPost and form element

### PR DESCRIPTION
**Pull request by smruthibalajee**
**Description:** Added comment section to webpage. Displays a form with three fields. Users can enter data into the form, which is then read and stored in an ArrayList. The data is then fetched and displayed under the form in a comment-style format. Have not added many CSS elements yet, but the nav bar currently goes to the right section.

**Key Features:** Includes checkboxes where the user can classify if they are a student, professional, etc. Then, this data is incorporated in their comment.

**Current progress (zoomed in):**
server-post

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -20,43 +20,68 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-import java.util.ArrayList;
+import java.util.*;
```
Comment (peter14f) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: nit: alphabetize the list of import statements

Comment (sabadas) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: I believe the Google Java style guide doesn't allow for wildcard imports

Comment (smruthibalajee) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: fixed!

Comment (smruthibalajee) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: got it, fixed!

**Review by peter14f — APPROVED** [review]

**Review by sabadas — APPROVED** [review]

**Pull request merged**